test_runner/regress/test_compaction.py

```diff
@@ -150,6 +150,57 @@ def test_pageserver_gc_compaction_smoke(neon_env_builder: NeonEnvBuilder):
     workload.validate(env.pageserver.id)
 
 
+@skip_in_debug_build("only run with release build")
+def test_pageserver_compaction_detach_ancestor_smoke(neon_env_builder: NeonEnvBuilder):
```

Comment:
nit: perhaps this test belongs in `test_timeline_detach_ancestor.py`?

Reply:
moved in https://github.com/neondatabase/neon/pull/9842/commits/60dec2cadfc7b400e81421d48b5a95a0bc4d38e4